src/jit/flowgraph.cpp

```diff
@@ -21523,6 +21527,35 @@ void                Compiler::fgInline()
     BasicBlock* block = fgFirstBB;
     noway_assert(block);
 
+    // Set the root inline context on all statements
+    // 
+    // Note we set ixlCode to nullptr here to preserve existing
+    // behavior, which is to allow one recursive inline.
+    inlExpLst* expDsc = new (this, CMK_Inlining) inlExpLst;
+    expDsc->ixlCode = nullptr;
+    expDsc->ixlParent = nullptr;
+    expDsc->ixlSibling = nullptr;
+    expDsc->ixlChild = nullptr;
+    expDsc->ilOffset = 0;
+
```

Comment:
Nit: Maybe default constructor to initialize these?
Shall we initialize ilOffset as BAD_IL_OFFSET although this root inline context doesn't need it?
Strictly speaking ixlSibling/ixlChild/ilOffset is not needed for release build. If we want to keep inline tree for other purpose in release build, we may want to retain method name to be useful.